core/src/main/java/org/infinispan/interceptors/distribution/VersionedDistributionInterceptor.java

```diff
@@ -30,7 +30,7 @@ protected Log getLog() {
    }
 
    @Override
-   protected void prepareOnAffectedNodes(TxInvocationContext ctx, PrepareCommand command, Collection<Address> recipients, boolean ignored) {
+   protected void prepareOnAffectedNodes(TxInvocationContext<?> ctx, PrepareCommand command, Collection<Address> recipients, boolean ignored) {
```

Comment:
Most users of `TxInvocationContext` ignore the type parameter, we can ignore it here too.
TBH I'm not sure why the context has a type parameter, I removed it and I didn't need to add any cast to make everything compile.